src/tmp.cpp

```diff
@@ -174,6 +172,18 @@ void remove(const fs::path& path) noexcept {
     }
 }
 
+/// Closes the given file, ignoring any errors
+/// @param file     The file to close
+void close(const file& file) noexcept {
+    if (!file.path().empty()) {
+#ifdef WIN32
+        CloseHandle(file.native_handle());
+#else
+        ::close(file.native_handle());
```

Comment:
```suggestion
        close(file.native_handle());
```
It was needed when close was a member function as it shadowed the global one. I think when they are both global, there should not be any problems with removing the leading `::`

Reply:
Still needed
```
/Users/bugdealer/Developer/tmp/src/tmp.cpp:182:9: error: no matching function for call to 'close'
        close(file.native_handle());
        ^~~~~
/Users/bugdealer/Developer/tmp/src/tmp.cpp:177:6: note: candidate function not viable: no known conversion from 'native_handle_type' (aka 'int') to 'const file' for 1st argument
void close(const file& file) noexcept {
     ^
```